content/articles/2018-12-textrecipes-0-0-1.Rmarkdown

```diff
@@ -0,0 +1,84 @@
+---
+title: 'textrecipes 0.0.1'
+author: Emil Hvitfeldt
+date: '2018-12-20'
+slug: textrecipes-0-0-1
+description: >
+    textrecipes 0.0.1 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/X1exjxxBho4
+  author: Roman Kraft
+---
+
+```{r setup, include = FALSE}
+library(textrecipes)
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're delighted to announce the release of [textrecipes 0.0.1](https://github.com/tidymodels/textrecipes) on CRAN. [`textrecipes`](https://github.com/tidymodels/textrecipes) implements a collection of new steps for the [`recipes`](https://github.com/tidymodels/recipes) package to deal with text preprocessing. [`textrecipes`](https://github.com/tidymodels/textrecipes) is still in early development so any and all feedback is highly appreciated.
+
+You can install it by running:
+
+```{r, eval = FALSE}
+install.packages("textrecipes")
+```
+
+## New steps
+
+The steps introduces here can be split into 3 types. The steps
+
+1. converts from characters to list-columns and vice versa
+1. modifies the elements in list-columns
+1. converts list-columns to numerics
+
+this allows for great flexibility in the preprocessing tasks that can be done while staying inside the [`recipes`](https://github.com/tidymodels/recipes) framework. This also prevents having one single step having a million arguments.
```

Comment:
* Start this as a new sentence (i.e. capitalise "this").
* Do you mean greater where you say great? If not, I'd strike the word.

> This also prevents having one single step having a million arguments.

Maybe change that second "having" to "with"? Also, I might switch out "a million" for something like "many."